src/OpenTelemetry.Api/OpenTelemetry.Api.csproj

```diff
@@ -9,7 +9,7 @@
     <Nullable>disable</Nullable>
   </PropertyGroup>
 
-  <ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0' or '$(TargetFramework)' == 'net462' or '$(TargetFramework)' == 'net6.0'">
```

Comment:
@thompson-tomo - The explicit dependency is intentional here and cannot be changed. We always want to depend on the latest `System.Diagnostics.DiagnosticSource` irrespective of tfm. Please revert this.

Reply:
Done.